content/applications/inventory_and_mrp/inventory/management/products/strategies.rst

```diff
@@ -1,75 +1,200 @@
+
+To create a new reordering rule from the replenishment report, go to :menuselection:`Inventory app
+--> Operations --> Replenishment`, click :guilabel:`Create`, and select the desired product from the
+drop-down menu in the :guilabel:`Product` column. If necessary, a :guilabel:`Min Quantity` and a
+:guilabel:`Max Quantity` can be configured in the corresponding columns on the replenishment report
+page, as well.
+
+To create a new reordering rule from the product form, go to :menuselection:`Inventory app -->
+Products --> Products`, and select a product to open its product form. Click the
+:guilabel:`Reordering Rules` smart button, click :guilabel:`Create`, and fill out the fields.
+
+Replenishment report fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following fields are on the :guilabel:`Replenishment` report. If any of these fields are not
+visible, click the additional options icon (three vertical dots) on the far right side of the
+report, then click the checkbox next to a field to make it visible.
+
+- :guilabel:`Product`: The product that requires a replenishment.
+- :guilabel:`Location`: The specific location where the product is stored.
+- :guilabel:`Wahrehouse`: The warehouse the product is stored.
+- :guilabel:`On Hand`: The amount of product currently available.
+- :guilabel:`Forecast`: The amount of product available after all current orders (sales,
+  manufacutirng, purchase, etc) are taken into account.
+- :guilabel:`Preferred Route`: How the product is procured, either :guilabel:`Buy`,
+  :guilabel:`Manufactured`, :guilabel:`Droship`, etc.
+- :guilabel:`Vendor`: The company the product is acquired from.
+- :guilabel:`Bill of Materials`: If a bill of materials is set for the product, it is listed here.
+- :guilabel:`Trigger`: How the replenishment is created, either :guilabel:`Auto` (automatically,
+  once the :guilabel:`On Hand` quantity goes below the :guilabel:`Min Quantity`) or
+  :guilabel:`Manual` (only when the replenishment is requested).
+- :guilabel:`Procurement Group`: The reference number for how the product is being acquired, such
+  as a sales order, purchase order, or manufacturing order.
+- :guilabel:`Min Quantity`: The minimum amount of product that should be available. When inventory
+  levels goes below this number, the replenishment is triggered.
+- :guilabel:`Max Quantity`: The amount of product that should be available after replenishing the
+  product.
+- :guilabel:`Multiple Quantity`: If the product should be ordered in specific quantities, enter the
+  number that should be ordered. For example, if the :guilabel:`Multiple Quantity` is set to 5, and
+  only 3 are needed, 5 products will be replenished.
+- :guilabel:`To Order`: The amount of product that is currently needed, and will be ordered, if the
+  :guilabel:`Order Once` or :guilabel:`automate Orders` button is clicked.
+- :guilabel:`UoM`: The unit of measure used to acquire the product.
+- :guilabel:`Company`: The company the product is acquired for.
```

Comment:
```suggestion
- :guilabel:`Product`: the product that requires a replenishment.
- :guilabel:`Location`: the specific location where the product is stored.
- :guilabel:`Wahrehouse`: the warehouse where the product is stored.
- :guilabel:`On Hand`: the amount of product currently available.
- :guilabel:`Forecast`: the amount of product available after all current orders (sales,
  manufacutirng, purchase, etc) are taken into account.
- :guilabel:`Preferred Route`: how the product is procured, either :guilabel:`Buy`,
  :guilabel:`Manufactured`, :guilabel:`Droship`, etc.
- :guilabel:`Vendor`: the company from which the product is acquired.
- :guilabel:`Bill of Materials`: the bill of materials for the product (if one is configured).
- :guilabel:`Trigger`: how the replenishment is created, either :guilabel:`Auto` (automatically,
  once the :guilabel:`On Hand` quantity goes below the :guilabel:`Min Quantity`) or
  :guilabel:`Manual` (only when the replenishment is requested).
- :guilabel:`Procurement Group`: the reference number for how the product is being acquired, such
  as a sales order, purchase order, or manufacturing order.
- :guilabel:`Min Quantity`: the minimum amount of product that should be available. When inventory
  levels goes below this number, the replenishment is triggered.
- :guilabel:`Max Quantity`: the amount of product that should be available after replenishing the
  product.
- :guilabel:`Multiple Quantity`: if the product should be ordered in specific quantities, enter the
  number that should be ordered. For example, if the :guilabel:`Multiple Quantity` is set to `5`,
  and only 3 are needed, 5 products are replenished.
- :guilabel:`To Order`: the amount of product currently needed, and will be ordered, if the
  :guilabel:`Order Once` or :guilabel:`Automate Orders` button is clicked.
- :guilabel:`UoM`: the unit of measure used to acquire the product.
- :guilabel:`Company`: the company for which the product is acquired.
```

hit the words following the colons with a "lower case," per ZST's preference - along with some minor grammatical adjustments